web/src/message_list_tooltips.js

```diff
@@ -266,19 +266,22 @@ export function initialize() {
         },
     });
 
-    message_list_tooltip([".recipient_bar_icon"], {
+    message_list_tooltip(".recipient_bar_icon", {
         delay: LONG_HOVER_DELAY,
         onHidden(instance) {
             instance.destroy();
         },
     });
 
-    message_list_tooltip([".rendered_markdown time", ".rendered_markdown .copy_codeblock"], {
-        content: timerender.get_markdown_time_tooltip,
-        onHidden(instance) {
-            instance.destroy();
+    message_list_tooltip(
+        [".rendered_markdown time", ".rendered_markdown .copy_codeblock"].join(","),
```

Comment:
Why is this required? I wonder if we just need to declare the type on `message_list_tooltip` better to pass-through the actual upstream type that supports string or array.

Reply:
This acts as `target` argument when calling  `tippyjs` `delegate` function. The `delegate` function accepts argument of type `tippy.Props` which has the `target` field of type `string`. Reference:https://github.com/atomiks/tippyjs/blob/ad85f6feb79cf6c5853c43bf1b2a50c4fa98e7a1/src/types.ts#L171

Reply:
OK, but this seems like a better solution, applied:

``` diff
diff --git a/web/src/message_list_tooltips.js b/web/src/message_list_tooltips.js
index 464fee44a2..ce2c05fc1c 100644
--- a/web/src/message_list_tooltips.js
+++ b/web/src/message_list_tooltips.js
@@ -273,8 +273,7 @@ export function initialize() {
         },
     });
 
-    message_list_tooltip(
-        [".rendered_markdown time", ".rendered_markdown .copy_codeblock"].join(","),
+    message_list_tooltip(".rendered_markdown time, .rendered_markdown .copy_codeblock",
         {
             content: timerender.get_markdown_time_tooltip,
             onHidden(instance) {
```

Reply:
Noted. Thanks !.